doc-content/enterprise-only/pages/building-custom-dashboard-widgets-running-heatmap-components-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='building-custom-dashboard-widgets-running-heatmap-components-proc_{context}']
+
+= Running heatmap components
```

Comment:
Hi @barbora ,
Can you please review this section

Reply:
Hey @sthundat can you please delete or edit this comment? I guess you have tagged the wrong Barbora here. The correct one is @barboras7. 

I am unable to remove @barbora from the thread.